prawns/governance/src/handlers/democracy.Proposed.event.ts

```diff
@@ -1,9 +1,35 @@
-import { EventHandlerContext } from '@subsquid/substrate-processor';
-import { decodeAddress } from '../utils';
-import { SubstrateNetwork, SubstrateDemocracyProposal } from '../model';
-import { getOrCreateGovernanceAccount } from '../utils';
+import { EventHandlerContext, ExtrinsicArg } from '@subsquid/substrate-processor';
+import { decodeAddress, getOrCreateGovernanceAccount } from '../utils';
+import { SubstrateDemocracyProposal, SubstrateNetwork } from '../model';
 import { getDemocracyProposedEvent } from './typeGetters/getDemocracyProposedEvent';
 
+const getProposalHash = (ctx: EventHandlerContext, network: SubstrateNetwork) => {
+
+  const args = ctx.event!.extrinsic!.args;
+
+  if (ctx.event.extrinsic?.method === 'batchAll') {
+    const calls = args[0].value as { args: {[key: string]: any} }[];
+    const batchProposeArgs = calls.find(call => call.args && call.args.proposal_hash);
```

Comment:
Is it possible to batch 2 of these extrinsic types? If so an event will be emitted for each, and the 1st extrinsic proposal hash will be applied in all the event handlers

Reply:
It's a good point. It's not been used before.. but I guess it might be possible?

Reply:
Added some defensive code so if it ever happens it will error.